newscoop/library/Newscoop/Entity/Repository/IssueRepository.php

```diff
@@ -60,13 +60,15 @@ public function getLatestByPublication($publicationId, $maxResults = 1)
         return $query;
     }
 
-    public function getByPublicationAndNumber($publication, $number)
+    public function getByPublicationAndNumber($publication, $number, $language)
```

Comment:
function name should be changed because its wrong and this function is used only in `SectionsController`